app/services/katello/ui_notifications/pulp/proxy_disk_space.rb

```diff
@@ -0,0 +1,39 @@
+module Katello
+  module UINotifications
+    module Pulp
+      class ProxyDiskSpace
+        class << self
+          def deliver!
+            SmartProxy.unscoped.with_features('Pulp').each do |proxy|
+              percentage = proxy.statuses[:pulp].storage['pulp_dir']['percent']
```

Comment:
maybe consider doing the percentage calculation in the ProxyApi:Pulp class.
(on the way you can unify pulp and pulpNode classes too:))